src/main/java/cinema/config/AppConfig.java

```diff
@@ -0,0 +1,50 @@
+package cinema.config;
+
+import cinema.model.CinemaHall;
+import cinema.model.Movie;
+import cinema.model.MovieSession;
+import cinema.model.Order;
+import cinema.model.ShoppingCart;
+import cinema.model.Ticket;
+import cinema.model.User;
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:db.properties")
+@ComponentScan(basePackages = {"cinema.service", "cinema.dao"})
+public class AppConfig {
+    @Autowired
+    private Environment environment;
+
+    @Bean
+    public DataSource getDataSource() {
+        BasicDataSource dataSource = new BasicDataSource();
+        dataSource.setUrl(environment.getProperty("db.url"));
+        dataSource.setDriverClassName(environment.getProperty("db.driver"));
+        dataSource.setUsername(environment.getProperty("db.username"));
+        dataSource.setPassword(environment.getProperty("db.password"));
+        return dataSource;
+    }
+
+    @Bean
+    public LocalSessionFactoryBean getSessionFactory() {
+        LocalSessionFactoryBean factory = new LocalSessionFactoryBean();
+        factory.setDataSource(getDataSource());
+        Properties properties = new Properties();
+        properties.put("hibernate.show_sql", environment.getProperty("hibernate.show_sql"));
+        properties.put("hibernate.hbm2ddl.auto", environment.getProperty("hibernate.hbm2ddl.auto"));
+        factory.setHibernateProperties(properties);
+        factory.setAnnotatedClasses(CinemaHall.class, Movie.class, MovieSession.class,
+                Order.class, ShoppingCart.class, Ticket.class, User.class);
```

Comment:
let's use setPackagesToScan() And you won't have to change your code every time while adding a new model